### DISCUSSION TITLE
ARN for the Amazon Kinesis data stream used for streaming your CTRs from the Amazon Kinesis Data streams console

### DISCUSSION THREAD

**Discussion by rongzhonghuang** [amazon-connect/amazon-connect-salesforce-scv#138] [Q&A]
Hi community, 

When I tried to get the ARN for the Amazon Kinesis data stream used for streaming your CTRs from the [Amazon Kinesis Data streams console](https://console.aws.amazon.com/kinesis/home), I don't see any data stream (please see the screenshot attached), am I missing any steps?  

<img width="1180" alt="Screen Shot 2022-09-28 at 10 29 12 am" src="https://user-images.githubusercontent.com/8980006/192660902-448d5e2d-684d-415d-93bc-3685a023346c.png">

**Comment by dougjaso** [comment]
Have you verified that you are in the right region?

**Comment by dougjaso** [comment]
With Service Cloud Voice, you will not see streams that are in an unsupported region. Make sure the appropriate region is selected. The Informational message in the screenshot indicates that this user is viewing resources in the correct region as the user is root and, in an SCV-built account, root is bound by the service control policy put in place by Salesforce.
